guides/Getting-Started.md

```diff
@@ -0,0 +1,77 @@
+# Getting Started With Test262
+
+
+Rough outline...
+
+
+- Why do we test?
+- What do we test? 
+  - Similar to https://github.com/tc39/test262/blob/main/CONTRIBUTING.md#acceptable-tests and https://github.com/tc39/test262/blob/main/CONTRIBUTING.md#unacceptable-tests
+- Anatomy of a test
+  - frontmatter
+    - esid
+    - description
+    - don't mention any other frontmatter
+  - test code
+    - assert api
+- Let's write a test, end to end.
+  - Pre-work
+    - Find something that needs to be tested
+    - Read spec
+    - Create test plan
+      - This can be a super minimal version of the usual test plans, but will help to show how we work
+        - Write a checklist (we'll give a 2-3 point example to work from)
+      - Does it need a new feature tag?
+    - Share test plan
+  - Writing our first test
+    - Git stuff
+      - Pull test272 repo
+      - Create a branch
+      - What else?
+    - Create a new file
+      - Where? Why?
+      - What do I name it? Why?
```

Comment:
Should we add the idea of how many tests per file here? Or maybe above in the create test plan section, something about "how to decompose behavior into files"?